### PR TITLE
fix: vulns

### PR DESCRIPTION
**Pull request by dkontorovskyy**
- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [X] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does
fix vulns

### CODE REVIEW

**Review by ivanstanev — COMMENTED** [review]
Amazing! Could we change the commit to "fix" or "feat" because we want to publish a new version of the app, with the fixed vulns? 🙏

**Review by agranado2k — DISMISSED** [review]
LGTM! 🎉

**Comment by dkontorovskyy** [comment]
@ivanstanev you right :) I still can't think properly after month off :)

File: package.json [hunk 1]
```diff
@@ -28,29 +28,29 @@
...
-    "snyk-config": "3.0.0",
+    "snyk-config": "3.1.1",
     "snyk-docker-plugin": "2.8.0",
     "source-map-support": "^0.5.16",
     "tunnel": "0.0.6",
     "typescript": "^3.8.3",
     "ws": "^7.2.1",
     "yaml": "^1.7.2",
-    "snyk": "^1.320.3"
+    "snyk": "^1.413.5"
```
Comment (shaimendel) [inline package.json:53]: assuming we have no patches - can't we remove this dependency and the .snyk file?

**Review by shaimendel — APPROVED** [review]

**Pull request merged**